setup.py

```diff
@@ -87,7 +94,9 @@ def get_secret_files() -> list:
         ('/opt/seagate/cortx/provisioner/conf', get_conf_files()),
         ('/opt/seagate/cortx/provisioner/conf', get_requirements_files()),
         ('/etc/cortx/solution/secret/', get_secret_files()),
-        ('/opt/seagate/cortx/provisioner/bin', ['src/setup/cortx_deploy'])
+        ('/opt/seagate/cortx/provisioner/bin', ['src/setup/cortx_deploy']),
+        ('/opt/seagate/cortx/k8s/bin', get_k8s_files()),
```

Comment:
Move to halo and not outside. 
/opt/seagate/halo/k8s/
/opt/seagate/halo/conf/

Reply:
done, but as provisioner looks for a defined path which is /opt/seagate/cortx/<component-name> ... as we discussed yesterday , i have added path in cluster.yaml now for reading custom path.